code/API_definitions/quality-on-demand.yaml

```diff
@@ -464,7 +464,6 @@ paths:
           application/json:
             schema:
               $ref: "#/components/schemas/RetrieveSessionsInput"
-        required: true
```

Comment:
There is a discussion about this within https://github.com/camaraproject/Commonalities/issues/247. What I'm getting from the discussion that the preferred option seems to make the structure not optional, even if it can be empty, see e.g. https://github.com/camaraproject/Commonalities/issues/247#issuecomment-2242341834

Reply:
I had my doubts about this but looking into the discussion I will revert it to required: true